### ISSUE TITLE
"Add a trigger" text visually appearing as heading is not defined as heading programmatically.

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#7281]
### Describe the issue

Repro:
1. go to edit workflow
2. select add a trigger

Actual Result:
 "Add a trigger" text visually appearing as heading is not defined as heading programmatically.
Expected Result:
 "Add a trigger" text visually appearing as heading is not defined as heading programmatically.

Solution: "Add a trigger" should be defined under <H2> tag.
MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

![Image](https://github.com/user-attachments/assets/6d664072-9edb-4ed6-85eb-6817869eb97b)

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Edge
